apps/demos/Demos/Toolbar/Adaptability/Vue/App.vue

```diff
@@ -451,4 +464,8 @@ function onFontFamilyClick() {
   height: 1px;
   border-bottom: 1px solid #ddd;
 }
+
+.dx-button-content {
```

Comment:
Why we should add this style in Vue only? As I can see there is the issue with aligning button's icon and text. This issue is reproducing for buttons that have `locateInMenu: always'.
<img width="230" alt="image" src="https://github.com/DevExpress/DevExtreme/assets/15689080/91869d2a-9e1d-49eb-a770-f6f0fad166f4">

Markup in Vue has particularities and list-item-content class present on the Button element. 
As for me, it looks like a bug. 

Well, we can leave it temporary solution for Vue demo, but let's create the Trello card with full description of the issue to consider this scenario in the nearest future.
We